test/jdk/javax/swing/JColorChooser/Test4887836.java

```diff
@@ -21,23 +21,40 @@
  * questions.
  */
 
+import java.awt.Color;
+import java.awt.Font;
+import javax.swing.JColorChooser;
+import javax.swing.UIManager;
+
 /*
  * @test
  * @bug 4887836
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
  * @summary Checks if no tooltip modification when no KeyStroke modifier
- * @author Konstantin Eremin
- * @run applet/manual=yesno Test4887836.html
+ * @run main/manual Test4887836
  */
+public class Test4887836 {
+
+    public static void main(String[] args) throws Exception {
+        String instructions = "If you do not see white area under the \"Swatches\" tab,\n" +
+                "then test passed, otherwise it failed.";
```

Comment:
The original instructions seem a bit vague and the test window has a lot going on so probably more details here would add clarity.

Reply:
The original test [JDK-4887836](https://bugs.openjdk.org/browse/JDK-4887836) is quite vague so I'm not sure how to add more details other than "white area under 'Swatches' tab"

Reply:
Which "White area" are they referring in Swatches tab? Is it the preview window?